src/main/kotlin/com/fatimacarneiro/estudokotlin/phonenumber/Dao.kt

```diff
@@ -0,0 +1,12 @@
+package com.fatimacarneiro.estudokotlin.phonenumber
+
+import org.springframework.data.jpa.repository.Query
+import org.springframework.data.repository.CrudRepository
+import org.springframework.stereotype.Repository
+
+@Repository
+interface Dao : CrudRepository<PhoneNumber, Int> {
+
+    @Query(value = "SELECT pn FROM PhoneNumber pn WHERE pn.ddd = :ddd")
```

Comment:
why do you need `@Query`?

Reply:
fun findByDdd(ddd: int): List<PhoneNumber>
fun findByDddAndNumber(ddd:Int, number:Long): Optional<PhoneNumber>